scalardb-analytics-postgresql-sample/README.md

```diff
@@ -60,22 +81,26 @@ erDiagram
     }
 ```
 
-`dynamons`, `postgresns`, and `cassandrans` are namespaces that are mapped to the back-end storages of DynamoDB, PostgreSQL, and Cassandra, respectively.
+For reference, this diagram shows the following:
 
-The `dynamons.customer` table represents information about customers. This table includes attributes like customer key, name, address, phone number, and account balance.
+- `dynamons`, `postgresns`, and `cassandrans`. Namespaces that are mapped to the back-end storages of DynamoDB, PostgreSQL, and Cassandra, respectively.
+- `dynamons.customer`. A table that represents information about customers. This table includes attributes like customer key, name, address, phone number, and account balance.
+- `postgresns.orders`. A table that contains information about orders that customers have placed. This table includes attributes like order key, customer key, order status, order date, and order priority.
+- `cassandrans.lineitem`. A table that represents line items associated with orders. This table includes attributes such as order key, part key, supplier key, quantity, price, and shipping date.
 
-The `postgresns.orders` table contains information about orders that customers have placed. This table includes attributes like order key, customer key, order status, order date, and order priority.
+### Schema details in PostgreSQL
 
-The `cassandrans.lineitem` table represents line items associated with orders. This table includes attributes such as order key, part key, supplier key, quantity, price, and ship date.
+By running the Schema Importer when setting up ScalarDB, you can import the table schema in the ScalarDB database into the PostgreSQL database. More precisely, for each `namespace_name.table_name` table in the ScalarDB database, you will have a foreign table for `namespace_name._table_name` and a view for `namespace_name.table_name` in the PostgreSQL database. The columns in the foreign table are identical to the columns in the table in PostgreSQL and in the transaction metadata.
```

Comment:
> The columns in the foreign table are identical to the columns in the table in PostgreSQL and in the transaction metadata.

I noticed this is a bit confusing me since I imagined the foreign table when hearing "the table in PostgreSQL". @choplin What does "the columns in the table in PostgreSQL" actually mean?

Reply:
This sentence is hard for me to understand what this sentence says... (although this was written by me, actually). Thank you for pointing this out.

@komamitsu @josh-wong How about the following?

> The created foreign table contains identical columns to the ScalarDB table as well as the transaction metadata columns that ScalarDB manages internally. The created view is defined to exclude the transaction metadata columns from the foreign table, so it only contains the same columns as the ScalarDB table.

Reply:
It seems far clearer to me 👍

Reply:
I see. Thank you for clarifying! 

Could you please clarify if `it` is referring to `foreign table`? If that's correct, I will change this paragraph as follows:

```suggestion
By running the Schema Importer when setting up ScalarDB, you can import the table schema in the ScalarDB database into the PostgreSQL database. More precisely, for each `namespace_name.table_name` table in the ScalarDB database, you will have a foreign table for `namespace_name._table_name` and a view for `namespace_name.table_name` in the PostgreSQL database. The created foreign table contains columns that are identical to the ScalarDB table and the transaction metadata columns that ScalarDB manages internally. Since the created view is defined to exclude the transaction metadata columns from the foreign table, the foreign table only contains the same columns as the ScalarDB table.
```

Reply:
IIUC, the foreign table and the view contain as follow
- the foreign table:
  - This contains the same columns as the underlying database table does
  - ScalarDB metadata (e.g., `tx_id`)
  - ScalarDB table columns (e.g., `customer_id`)
    - I'm not sure what is a correct term for this
- the view
  - Only ScalarDB table columns (e.g., `customer_id`)

> so it only contains the same columns as the ScalarDB table

I think `it` means `the created view` which doesn't contain the metadata.

Reply:
Yes, `it` refers to the `the created view` in the I proposed. Thank you @komamitsu for clarifying it. So, the last sentence in your suggestion should be corrected as `the created view only contains the same columns as the ScalarDB table`.

Reply:
BTW, I wonder whether it is worth explaining the details of the foreign table because most of the users would be interested only in the created views.

Reply:
Thank you for clarifying! I've modified the sentence according to your feedback.

> BTW, I wonder whether it is worth explaining the details of the foreign table because most of the users would be interested only in the created views.

@choplin Do you think we need to remove a part?

Reply:
Thanks! Please keep the part for now. After we have more users, we will get an insight into whether it is actually helpful for users.